config/routes.rb

```diff
@@ -30,21 +30,23 @@
 
   get "/about"                => "home#about"
 
-  get "/admin/organizations"  => "admin#organizations",
-                              as: :admin_organizations
+  namespace :admin do
+    get "/organizations"  => "admin#organizations",
+                                as: :organizations
```

Comment:
Please convert those routes to using something like `resources :organizations` and limit them to the actions that are used.